<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>607</width>
    <height>241</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="Button">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>20</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Расчёт</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditA">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>71</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditC">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>20</y>
      <width>71</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditB">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>20</y>
      <width>71</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="Alabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>a</string>
    </property>
   </widget>
   <widget class="QLabel" name="Blabel">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>20</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>b</string>
    </property>
   </widget>
   <widget class="QLabel" name="Clabel">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>20</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>c</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditResult">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>110</y>
      <width>401</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="Reslabel">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>80</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Результат: </string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>607</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
